test/suites/common/setup.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: setup
+  namespace: karpenter-tests
+spec:
+  description: |
+    Execute a karpenter test suite
```

Comment:
It's unclear to me when we'd use `setup.yaml` vs `install-karpenter.yaml`. Can you update the descriptions of these files to explain this? Would this be used to create an EKS cluster and install karpenter, and `install-karpenter.yaml` would be used on an existing cluster? If that's the case, is there use in de-duplicating this into a `create-eks.yaml` and `install-karpenter-eks.yaml` like in the examples?

Reply:
Meant to delete install-karpenter. It's deleted now.